dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java

```diff
@@ -390,6 +391,8 @@
     K8S_CLIENT_OPS_ERROR(1300006, "k8s error with exception {0}", "k8s操作报错[{0}]"),
     VERIFY_K8S_NAMESPACE_ERROR(1300007, "verify k8s and namespace error", "验证k8s命名空间信息错误"),
     DELETE_K8S_NAMESPACE_BY_ID_ERROR(1300008, "delete k8s namespace by id error", "删除命名空间错误"),
+    VERIFY_PARAMETER_NAME_FAILED(503,"The file name verify  failed","文件命名校验失败"),
+    STORE_OPERATE_CREATE_ERROR(504,"create the resource failed","存储操作失败")
```

Comment:
Should we avoid using an error code the same as exists/default HTTP code?

Reply:
i changed thems to 130009 and 1300010